packages/api/src/command/medical/patient/create-patient.ts

```diff
@@ -42,5 +44,12 @@ export const createPatient = async (patient: PatientCreateCmd): Promise<Patient>
   // Intentionally asynchronous - it takes too long to perform
   cwCommands.patient.create(newPatient, facilityId).catch(processAsyncError(`cw.patient.create`));
 
+  // TODO: NEED CQ PATIENT DIRECTORY SEARCH
+  // const organization = await getOrganizationOrFail({ cxId, id: facilityId });
+
+  // cqCommands.patient
+  //   .discover(newPatient, organization, facilityId)
+  //   .catch(processAsyncError(`cq.patient.create`));
+
   return newPatient;
```

Comment:
This is still a work in progress as it requires the cq directory patient facility locator logic. Also we dont want this logic going into the cloud until the ihe gateway infra is ready. 

This was more for me to wrap my head around the whole flow

Reply:
This will probably be better to talk about in engineering

Reply:
Regardless of what we do with FF, let's tie this TODO into a specific issue? ie "Add CQ XCPD on Patient Create"

For the FF, I've added it in the Go Live checklist https://github.com/metriport/metriport-internal/issues/1350

Reply:
Should i create that ticket and then move those changes into a new branch so we dont lose them. That way i can clean this up